sdk/communication/AzureCommunicationChat/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+## 1.2.0-beta.1 (unreleased)
+### New Features
+- `ChatClient` now supports Push Notifications for Chat events
+- Following methods added to `ChatClient`:
+  - `startPushNotifications(deviceToken:encryptionKeyPair:)`
+  - `stopPushNotifications()`
+  - `decryptPayload(notification:encryptionKeyPairs:)`
+
```

Comment:
This is going to need to be 1.3.0-beta.1 because Lu Chen is in the process of releasing 1.2.0

Reply:
Already fixed it.